zebra-consensus/Cargo.toml

```diff
@@ -59,8 +59,9 @@ tracing = "0.1.39"
 tracing-futures = "0.2.5"
 
 orchard = "0.6.0"
+sapling = { package = "sapling-crypto", git = "https://github.com/oxarbitrage/sapling-crypto", branch = "add-verifying-key-methods" }
```

Comment:
This is the custom `sapling-crypto` version, specifically i am using custom inner functions to get inner types, instead we need to to something as described in https://github.com/zcash/sapling-crypto/issues/136
By doing it this way we are skipping type safety so i don't think upstream will allow the way i did it here but i think it helps to show the goal.